overlord/snapstate/snapstate.go

```diff
@@ -52,9 +52,12 @@ const (
 	// for use in the interim time while we have the backward compatible
 	// support
 	firstInterimUsableFlagValue
-	// if we need flags for just SnapSetup it may be easier
-	// to start a new sequence from the other end with:
-	// 0x40000000 >> iota
+)
+
+// SnapSetup specific flags
+const (
+	// RollbackOp means the given snap got rolled back
+	RollbackOp = 0x40000000 >> iota
```

Comment:
same comment as I mentioned in first boot, given that this is only for SnapSetup flags it can be typed as SnapSetupFlags